cypress/e2e/assets_spec/asset_homepage.cy.ts

```diff
@@ -27,6 +27,7 @@ describe("Asset Tab", () => {
 
   beforeEach(() => {
     cy.restoreLocalStorage();
+    cy.clearLocalStorage(/filters--.+/);
```

Comment:
I doubt if this is a good hack 😅

1. It's not maintainable in the long run.
2. Could give results that are not predictable.

We could tackle this in a separate issue, maybe by having a "Clear all" filter button on the page itself or by modifying existing cypress tests to adapt to this change.